### ISSUE TITLE
[rfc] - WAND lock control loop gain

### ISSUE THREAD

**Issue by Jemetzner** [OxfordIonTrapGroup/wand#39]
After reviewing the lock method code have discovered that the control loop has the gain set by the product of the set gain and the poll time. This would seem to suggest this is a integrative term, but would only seem to make sense in the limit of a very slowly converging lock.

I have calibrated our laser controllers to get a proper gain factor and turned this control loop into a simple proportional term by removing this poll time factor. We were hoping that there might be some clarification for why this was the method that was used and if there is a interest to improve the lock to make a proper integrator term.

**Comment by hartytp** [comment]
> This would seem to suggest this is a integrative term, but would only seem to make sense in the limit of a very slowly converging lock.
> I have calibrated our laser controllers to get a proper gain factor and turned this control loop into a simple proportional term by removing this poll time factor. We were hoping that there might be some clarification for why this was the method that was used and if there is a interest to improve the lock to make a proper integrator term.

Yes, currently there is only an integrator, designed for relatively slow feedback loops. 

https://github.com/OxfordIonTrapGroup/wand/blob/492e0ba97ba1ba4eef3c11105d25e4bbf76c84ab/wand/frontend/wand_server.py#L237

I don't think I understand your question. Can you explain what you think a "proper integrator term" and "simple proportional term"  would look like in this context?
